### ISSUE TITLE
Deps upgrade package_info_plus to 7.0.0

### ISSUE THREAD

**Issue by marwenbk** [getsentry/sentry-dart#1974]
### Description

upgrade package_info_plus to ^7.0.0

**Comment by betimer** [comment]
Any update on this? Current version is 7.0.0, where it askes ^4.2.0

```
* Consider downgrading your constraint on package_info_plus: dart pub add package_info_plus:^4.2.0
```

**Comment by aggeloskoutanis** [comment]
I need this one too

**Comment by ueman** [comment]
Use the following until the version with the fix is rolled out
```yaml
dependency_overrides:
  package_info_plus: ^7.0.0
```

The fix is already merged but not released:

https://github.com/getsentry/sentry-dart/blob/c687d6477079c3ccd00e9b0bd4e8aa52038f750a/flutter/pubspec.yaml#L19

(This is a duplicate of https://github.com/getsentry/sentry-dart/issues/1944 )

**Comment by buenaflor** [comment]
~Closing as duplicate (the fix is out as of `7.19.0`)~

**Comment by kuhnroyal** [comment]
This is actually not fixed, since there is a 7.0.0 out and the current upper range is `<7.0.0` :)

**Comment by buenaflor** [comment]
Ah my bad, didn't realize it was about 7 and not 6 😅, nvm then
